src/fidesops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -86,9 +86,15 @@ def get_connections(
     db: Session = Depends(deps.get_db),
     params: Params = Depends(),
     search: Optional[str] = None,
+    disabled: Optional[bool] = None,
+    connection_type: Optional[List[str]] = Query(default=None),  # type:ignore
```

Comment:
to search for multiple connection_types do: `?connection_type=postgres&connection_type=mongo`.  

I would have preferred this syntax: `?connection_type=postgres,mongo` for an OR query, but I am using the syntax in the docs https://fastapi.tiangolo.com/it/tutorial/query-params-str-validations/#query-parameter-list-multiple-values which works well with the autogenerated API docs:

<img width="594" alt="Screen Shot 2022-06-16 at 6 33 51 PM" src="https://user-images.githubusercontent.com/9755598/174195107-9956051f-697e-48b0-af17-7e054ca1d7e1.png">